extension/README.md

```diff
@@ -96,6 +96,24 @@ user's machine, you should build them yourself for each supported platform and
 include them in the .vsix or host them on a server and have your extension
 download them at runtime.
 
+### OS specific Extensions
+
+Use optional `osSpecificVsix` to choose a .vsix file by the [operating system](https://nodejs.org/api/os.html#os_os_platform):
```

Comment:
Minor: it would be nice to explain why `osSpecificVsix` is optional. Maybe something like this?

> By default, the first .vsix file in the `files` array is used. If you have different native dependencies for each platform, you can use `osSpecificVsix` to choose a .vsix file by the operating system.